### PR TITLE
ci: Updates

### PR DESCRIPTION
**Pull request by nhooyr**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@
 node_modules
 build
 dist
+play.d2lang.com
```
Comment (alixander) [inline .gitignore:7]: what's this

Comment (nhooyr) [reply .gitignore:7]: ci.sh builds the binary.

This is from when the module name was different, will update

**Review by alixander — APPROVED** [review]

File: make.sh [hunk 2]
```diff
@@ -6,7 +6,11 @@ if [ ! -e "$(dirname "$0")/ci/sub/.git" ]; then
...
 . "$(dirname "$0")/ci/sub/lib.sh"
-PATH="$(cd -- "$(dirname "$0")" && pwd)/ci/sub/bin:$PATH"
 cd "$(dirname "$0")"
 
-_make "$@"
+job_parseflags "$@"
+runjob fmt ./ci/sub/bin/fmt.sh &
+runjob lint ci_go_lint &
+runjob build 'go build ./...' &
+runjob test 'go test ./...' &
```
Comment (alixander) [inline make.sh:15]: there is not test

**Review by alixander — COMMENTED** [review]
why the deviation from others? (that use Makefile)

File: make.sh (hunk 2 above)
Comment (nhooyr) [reply make.sh:15]: No harm in leaving this for when there are tests. Just the template.

**Comment by nhooyr** [comment]
@alixander Make sucks, this uses my make style job.sh library. It has much better targeting of jobs with a go style regex. Please see the PR I made here https://github.com/terrastruct/text-to-diagram-site/pull/33 for specific examples of its use.

And you can better control output when a job doesn't need to run whereas with make it involves setting an environment variable in `make.sh` before running the Makefile. See https://github.com/terrastruct/src/blob/master/make.sh#L14

As well, see `bin/fmt.sh` in the CI repo for how much cleaner it makes things. See https://github.com/terrastruct/ci/commit/37d4202ed8bb057ff3cd1f1cd6ab85a205a5f189. Compare to the the code it replaced.

**Pull request merged**